README.md

```diff
@@ -71,19 +100,18 @@ Attribute              | Type          | Default                | Description
 `retransmissionThreshold`| *Number*    | `3`                    | After server reports this number of messages missing, we start retransmission
 `onReconnectionCountdown`| *Function*  |                        | Triggered when palindrom detected connection problem and reconnection is scheduled. Accepts number of milliseconds to scheduled reconnection. Called every second until countdown reaches 0 (inclusive)
 `onReconnectionEnd`    | *Function*    |                        | Triggered when palindrom successfully reconnected
-`jsonpatch`            | *Object*      | `window.jsonpatch`       | The provider object for jsonpatch apply, validate, observe and unobserve. By default assumes Starcounter-Jack/JSON-Patch library available in global `jsonpatch` variable.
+`jsonpatch`            | *Object*      | `window.jsonpatch`       | The provider object for jsonpatch `apply` and  `validate`. By default it uses Starcounter-Jack/JSON-Patch library.
 
 most of them are accessible also in runtime:
 
 #### Properties
 
-```javascript
+```js
 palindrom.property
 ```
 Attribute             | Type       | Default                | Description
 ---                   | ---        | ---                    | ---
-`remoteUrl`           | *String*   | `window.location.href` | See above
-`obj`                 | *Object*   | `{}`                   | See above
```

Comment:
"You can't" or "you shouldn't" Technically it is possible. So if it's our limitation it's yet another reason to have it described in the README.md. Maybe we should change it in the code to just a getter?